packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -156,6 +158,26 @@ export class DocumentDeltaConnection
 				0x20b /* "mismatch" */,
 			);
 			if (!this.trackedListeners.has(event)) {
+				if (event === "pong" && this.trackLatencyTimer === undefined) {
+					// log latency every minute
+					this.trackLatencyTimer = setInterval(() => {
+						const start = Date.now();
+
+						// emit pong event every minute. If latency is longer than 1 min, log separately
+						this.socket.emit("pong", () => {
```

Comment:
Let's use "ping". "pong" is a response to "ping" and here we are sending original signal, so "ping" name seems like more appropriate.
Appropriate server changes should happen too, as it response to this message.